### DISCUSSION TITLE
more harversters/tractors working together on one field at the same time

### DISCUSSION THREAD

**Discussion by Edwin1631** [Courseplay/Courseplay_FS22#3567] [Q&A] [answered]
Hello,
Before you could select how many harversters/tractors you wanted to work on a field.. you made a course for one harverster/tractor, selected 2 harversters and loaded them up in both machines and then selected for one left and the other one right.. but i cant find that option anymore after the update..

**Comment by Edwin1631** [comment]
its called multiple tools https://www.youtube.com/watch?v=UBHXSWpusZE

**Comment by Tensuko** [comment, marked as answer]
https://courseplay.github.io/Courseplay_FS22.github.io/

I know how it is called.
Read the help menu....

It is put into expert mode (turn it on in the global settings).

**Reply by Edwin1631** [reply]
thank you, i didn"t look that good.
